connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigBackingStore.java

```diff
@@ -901,6 +947,19 @@ public void onCompletion(Throwable error, ConsumerRecord<String, byte[]> record)
                 if (request != null && started) {
                     updateListener.onRestartRequest(request);
                 }
+            } else if (record.key().startsWith(TASK_COUNT_RECORD_PREFIX)) {
+                String connectorName = record.key().substring(TASK_COUNT_RECORD_PREFIX.length());
+                if (!(value.value() instanceof Map)) {
+                    log.error("Found task count record ({}) in wrong format: {}",  record.key(), value.value().getClass());
```

Comment:
We should include the connector name, I think.

Reply:
It's implicitly included in the record key but that info is redundant. Updated to just use the connector name and make the message clearer